script/build-linux

```diff
@@ -5,7 +5,7 @@ set -ex
 ./script/clean
 
 TAG="docker-compose"
-docker build -t "$TAG" .
+docker build -t "$TAG" . | tail -n 200
```

Comment:
Is this causing a problem? Might risk us losing useful debugging output if there is an issue...